aggregators/1.0/modules/ROOT/pages/aggregators-troubleshooting.adoc

```diff
@@ -0,0 +1,112 @@
+= Troubleshoot Aggregators Module - Mule 4
+
+To troubleshoot Aggregators module, become familiar with the information about app logs, enabling verbose logging, and interpreting commonly thrown messages.
+
+== Use the Latest Aggregators Module Version
+
+To ensure that you use the latest patches, update the Aggregator module to the latest version. In the Apache Maven `pom.xml` file generated by Anypoint Studio replace `x.x.x` with the version that corresponds to the latest version:
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>org.mule.modules</groupId>
+  <artifactId>mule-aggregators-module</artifactId>
+  <version>x.x.x</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+To obtain the most up-to-date `pom.xml` file information:
+
+. Go to https://www.mulesoft.com/exchange/[Anypoint Exchange].
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for `aggregators`.
+. Select the connector.
+. Click *Dependency Snippets* near the upper right of the screen.
+
+
+== Enable Verbose Logging
+
+To begin troubleshooting Aggregator module, enable verbose logging to see the exact error messages:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
```

Comment:
You don't "open" a name, so use either "open your app project" or "click your app project's name."